examples/vulkan/Vulkan.cpp

```diff
@@ -871,14 +865,14 @@ class VulkanExample
             return;
         }
 
-        swapchainExtent.width  = clamp<std::uint32_t>(window.getSize().x,
-                                                     surfaceCapabilities.minImageExtent.width,
-                                                     surfaceCapabilities.maxImageExtent.width);
-        swapchainExtent.height = clamp<std::uint32_t>(window.getSize().y,
-                                                      surfaceCapabilities.minImageExtent.height,
-                                                      surfaceCapabilities.maxImageExtent.height);
+        swapchainExtent.width  = std::clamp<std::uint32_t>(window.getSize().x,
```

Comment:
Is the template argument required?

Reply:
Yes it's required unless we change that `2` to a `2u` which I think I'd prefer.

Reply:
I was able to remove them all.